### DISCUSSION TITLE
Suggestions on CS9 + kernel-ml package

### DISCUSSION THREAD

**Discussion by memnochproxy** [openzfs/zfs#15032] [Q&A]
I have a CentOS Stream 9 server with kernel 6.1.8 that has a broken zfs userspace. This is a Ryzen system and the motherboard and cpu combination work better under 6.1 than the 5.x kernels. 
```shell
nf install zfs-dracut-2.1.12-1.el9.noarch
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered with an entitlement server. You can use subscription-manager to register.

Last metadata expiration check: 3:42:19 ago on Mon 03 Jul 2023 03:18:07 PM PDT.
Error: 
 Problem: package zfs-2.1.12-1.el9.x86_64 from zfs requires zfs-kmod = 2.1.12, but none of the providers can be installed
  - package zfs-dracut-2.1.12-1.el9.noarch from zfs requires zfs >= 2.1.12, but none of the providers can be installed
  - package zfs-dkms-2.1.12-1.el9.noarch from zfs requires kernel-devel >= 3.10, but none of the providers can be installed
  - package zfs-dkms-2.1.12-1.el9.noarch from zfs requires kernel-devel <= 6.3.999, but none of t
``` 
Besides cloning the git repo and re-compiling for every kernel I get, is there a way I can stay up to date with zfs rather than stick on zfs 2.1.9.

**Comment by rincebrain** [comment]
The easy way would seem to be to add a kernel-devel|kernel-ml-devel for that, but ideally it should have some virtual package it provides that could be depended on consistently.

Ideally and reality, of course, may differ.
